frontend/src/screens/subwallets/SubwalletList.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  HandCoins,
+  HelpCircle,
+  Landmark,
+  TriangleAlert,
+  Wallet2,
+} from "lucide-react";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import AppHeader from "src/components/AppHeader";
+import AppCard from "src/components/connections/AppCard";
+import ExternalLink from "src/components/ExternalLink";
+import Loading from "src/components/Loading";
+import { Button } from "src/components/ui/button";
+import { Input } from "src/components/ui/input";
+import { Label } from "src/components/ui/label";
+import { LoadingButton } from "src/components/ui/loading-button";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "src/components/ui/tooltip";
+import { useToast } from "src/components/ui/use-toast";
+import UpgradeCard from "src/components/UpgradeCard";
+import { UpgradeDialog } from "src/components/UpgradeDialog";
+import { useAlbyMe } from "src/hooks/useAlbyMe";
+import { useApps } from "src/hooks/useApps";
+import { useInfo } from "src/hooks/useInfo";
+import { createApp } from "src/requests/createApp";
+import { CreateAppRequest } from "src/types";
+import { handleRequestError } from "src/utils/handleRequestError";
+
+export function SubwalletList() {
+  const navigate = useNavigate();
+  const [name, setName] = React.useState("");
+  const { data: apps } = useApps();
+  const { toast } = useToast();
+  const { data: albyMe, error: albyMeError } = useAlbyMe();
+  const { data: info } = useInfo();
+  const onboardedApps = apps
+    ?.filter((app) => app.metadata?.app_store_app_id === "uncle-jim")
+    .sort((a, b) => (a.createdAt < b.createdAt ? 1 : -1));
+
+  const subwalletAmount = onboardedApps?.reduce(
+    (total, app) => total + app.balance,
+    0
+  );
+
+  const [isLoading, setLoading] = React.useState(false);
+  const [showIntro, setShowIntro] = React.useState(true);
+  const showForm =
+    albyMe?.subscription.plan_code ||
+    (onboardedApps && onboardedApps?.length < 3);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setLoading(true);
+
+    try {
+      if (apps?.some((existingApp) => existingApp.name === name)) {
+        throw new Error("A connection with the same name already exists.");
+      }
```

Comment:
We have now changed the backend to prefix (x) to the name if it already exists, so we can remove this

Can you also remove it in `SupportAlby.tsx` as well?